alma/alma-installments-prestashop
# Feat/mpp 509/add spinner during payment modal remove unmout in close modal

### Reason for change



[Loader + remove unmount close modal](https://linear.app/almapay/issue/MPP-509/add-spinner-during-payment-modal-remove-unmout-in-close-modal)

Add loader during payment modal + Remove unmount on close modal

### Code changes



### How to test

_As a reviewer, you are encouraged to test the PR locally._



Make a payment with inpage
And open and close modal to see if the payment is alway mount for the same payment

### Checklist for authors and reviewers




### Non applicable
